tock/projects/models.py

```diff
@@ -247,6 +247,11 @@ class Project(models.Model):
         default=False,
         help_text='Check if this project should be excluded from calculations of billable hours, e.g. Out of Office'
     )
+    include_in_utilization = models.BooleanField(
+            default=False,
+            help_text="For handling microrequests, per github.com/18F/tock/issues/1084"
```

Comment:
There's an opportunity here to use the `help_text` attribute to describe the functionality that setting this field to `True` provides. Noting the origin in the code does provide much utility.

Reply:
addressed in a9259f2